### DISCUSSION TITLE
GitHub Education benefits

### DISCUSSION THREAD

**Discussion by MattiaCalzavara** [community/community#132143] [GitHub Education]
### Hi everyone,

Hi, my GitHub Education benefits were approved more than 3 days ago, but I haven't received any further information yet.

**Comment by ghost** [comment]
Hi, you are not the only one, some people have exactly the same situation, I recently read that one person had them only after 9 days, try to wait a few more days

**Comment by davevad93** [comment]
Hi @MattiaCalzavara ,

It usually takes a few days, but sometimes it can even take a couple of weeks; the waiting time may vary depending on different factors, e.g. the volume of applications.

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @MattiaCalzavara was approved and your benefits are now available.`

**Comment by danialtavakoli** [comment]
If your GitHub Education benefits were approved more than 3 days ago and you haven't received any further information yet, here are a few steps you can take:

1. **Check Your Email**: Make sure to check the email associated with your GitHub account, including your spam or junk folder. GitHub typically communicates important updates via email.

2. **GitHub Student Developer Pack Dashboard**: Log in to your GitHub account and visit the GitHub Student Developer Pack dashboard. This is where you can find information about your approved benefits and any next steps you might need to take.

3. **Contact GitHub Support**: If you haven't received any emails or updates in the dashboard, consider reaching out to GitHub Support directly. They can provide you with specific information about the status of your application and any potential issues that might be delaying the process.

4. **Patience**: Sometimes, there might be delays due to high volumes of requests or internal processing issues. If this is the case, it's best to wait a little longer or contact support for clarification.

Here is the link to GitHub Support where you can reach out for further assistance: [GitHub Support](https://support.github.com/contact)

By following these steps, you should be able to get more clarity on the status of your GitHub Education benefits application.
